### ISSUE TITLE
missing metadata.json

### ISSUE THREAD

**Issue by vchepkov** [electrical/puppet-lib-file_concat#7]
puppet module command requires metadata.json file:
```
# puppet module list
Warning: Missing dependency 'ispavailability-file_concat':
  'elasticsearch-logstash' (v0.5.1) requires 'ispavailability-file_concat' (>= 0.1.0)
/etc/puppetlabs/puppet/environments/production/modules
├── elasticsearch-elasticsearch (v0.3.2)
├── elasticsearch-logstash (v0.5.1)
├── file_concat (???)
```

**Comment by electrical** [comment]
Its on my todo list to update this module and add the metadata.json file.
Hope to do it soon.
Thanks for reporting it.

**Comment by electrical** [comment]
This has been fixed with #11
